server/datastore/mysql/apple_mdm.go

```diff
@@ -140,6 +140,20 @@ func (ds *Datastore) DeleteMDMAppleConfigProfile(ctx context.Context, profileID
 	return nil
 }
 
+func (ds *Datastore) DeleteMDMAppleConfigProfileByTeamAndIdentifier(ctx context.Context, teamID uint, profileIdentifier string) error {
+	res, err := ds.writer.ExecContext(ctx, `DELETE FROM mdm_apple_configuration_profiles WHERE team_id = ? AND identifier = ?`, teamID, profileIdentifier)
+	if err != nil {
+		return ctxerr.Wrap(ctx, err)
+	}
+
```

Comment:
Unrelated to the PR, but those (new?) codecov warning are a bit much. Hopefully we can figure out a way to disable them, especially as we have a lot of code coverage from integration tests, which will not show up as covered for lines outside the service package.